network/containerizer/bridgepolicy.go

```diff
@@ -188,6 +189,25 @@ var skippedDeviceNames = set.NewStrings(
 	network.DefaultKVMBridge,
 )
 
+// The general policy is to:
+// 1. Add br- to device name (to keep current behaviour), if it doesn fit in 15 characters then:
+// 2. Add b- to device name, if it doesn't fit in 15 characters then:
```

Comment:
AFAIK USB device names are constructed with MAC - enx00e07cc81e1d, we have a test for that.